backend/test/session.test.js

```diff
@@ -1,21 +1,63 @@
 import request from 'supertest';
-
 import app from '../src/app';
-describe('sessions', () => {
-  it('should respond with 200 - OK ', done => {
-    request(app)
+import { UserRepository } from '../src/repository/UserRepository';
+import jwt from 'jsonwebtoken';
+
+jest.mock('../src/repository/UserRepository');
+jest.mock('jsonwebtoken');
+
+describe('Testing /api/session endpoint ', () => {
+  it('It should response with an error ', async () => {
+    const InvalideUser = { username: 'Lehel' };
+    const userResult = {
+      id: 1,
+      name: 'Lehel',
+      email: 'test@test.com',
+      password: 'password123',
+      isAdmin: 1,
+    };
+    const response = { results: [userResult] };
+
+    UserRepository.prototype.getUser.mockImplementation(() =>
+      Promise.resolve(response)
+    );
+    jwt.sign.mockImplementation((data, sec) => {
+      return data;
+    });
+
+    await request(app)
       .post('/api/session')
       .set('Content-Type', 'application/json')
-      .send({ name: 'Lehel', password: 'asdf' })
+      .send(InvalideUser)
```

Comment:
@LLaszlo89 either fix the spelling and variable names start with lowerCase letters or just put here the object